addons/mail/static/src/core/public_web/thread_model_patch.js

```diff
@@ -38,7 +38,11 @@ patch(Thread.prototype, {
                         !this.store.discuss.isActive &&
                         this.store.chatHub.opened.length < this.store.chatHub.maxOpened
                     ) {
-                        chatWindow.open();
+                        chatWindow.open({
+                            focus: !this.store.chatHub.opened.some(
+                                (cw) => cw.thread.composer.isFocused
+                            ),
+                        });
```

Comment:
This is a really strange condition.
This should just be always false.
Focus should not change from something that is not coming from an action of current user.
Especially not receiving a message on the bus.

Reply:
I don't get it, `isFocused` is true when a composer has the focus.
If a composer as the focus already we don't "steal" it. Otherwise, we can set the focus on the new chat window.

I don't see how the bus is involved.

Reply:
What's not working if this is just `false`?

This function is only called inside new message bus notif.

Reply:
If it already has focus, normally there is no need to focus again?

Reply:
If another composer has focus, this will steal the focus from the other composer to the one of channel with the new message, it makes no sense.